### PR TITLE
Feature/improve twitter sentiment app

### PR DESCRIPTION
**Pull request by anwar6953**
Various improvements to the TwitterSentiment App: …
-simplify logic in Normalization Flowlet.
-route the TweetCollector directly to the PythonAnalysisFlowlet.
-rename flowlets to be more informative and consistent with our naming convention.
-remove 'withResources' in flowBuilder, procedureBuilder. reason: deprecated.
-remove package-info.java
-@Batch processing in flowlet: Iterator<Tweet>   ->   Tweet
-fix format of pom.xml, move version into properties

### CODE REVIEW

File: TwitterSentiment/src/main/java/co/cask/cdap/apps/sentiment/TweetParserFlowlet.java [hunk 1]
```diff
@@ -25,22 +24,16 @@
 import co.cask.cdap.api.flow.flowlet.OutputEmitter;
 import co.cask.cdap.api.flow.flowlet.StreamEvent;
 import co.cask.cdap.api.metrics.Metrics;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
-import java.util.Map;
 
 /**
  * Normalizes the sentences.
```
Comment (abaranau) [inline TwitterSentiment/src/main/java/co/cask/cdap/apps/sentiment/TweetParserFlowlet.java:29]: update javadocs everywhere where you changed stuff

File: TwitterSentiment/src/test/java/co/cask/cdap/apps/sentiment/TwitterSentimentAppTest.java [hunk 2]
```diff
@@ -59,7 +59,7 @@ public void testSentimentProcedure() throws Exception {
         streamWriter.send("i am happy today that I got this working.");
 
         // Wait for the last Flowlet processed all tokens
-        RuntimeMetrics countMetrics = RuntimeStats.getFlowletMetrics(TwitterSentimentApp.APP_NAME, SentimentAnalysisFlow.FLOW_NAME, Update.UPDATE_FLOWLET_NAME);
+        RuntimeMetrics countMetrics = RuntimeStats.getFlowletMetrics(TwitterSentimentApp.APP_NAME, SentimentAnalysisFlow.FLOW_NAME, CountSentimentFlowlet.NAME);
```
Comment (abaranau) [inline TwitterSentiment/src/test/java/co/cask/cdap/apps/sentiment/TwitterSentimentAppTest.java]: SentimentAnalysisFlow.FLOW_NAME -> SentimentAnalysisFlow.NAME to make consistent

**Comment by abaranau** [comment]
please address minor comments and merge

File: TwitterSentiment/src/main/java/co/cask/cdap/apps/sentiment/TweetParserFlowlet.java (hunk 1 above)
Comment (anwar6953) [inline TwitterSentiment/src/main/java/co/cask/cdap/apps/sentiment/TweetParserFlowlet.java:29]: README already updated:
https://github.com/caskdata/cdap-apps/pull/27/files#diff-1

**Comment by abaranau** [comment]
LGTM!

**Pull request merged**